androidApp/src/main/res/values/strings.xml

```diff
@@ -1,17 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
-<resources xmlns:tools="http://schemas.android.com/tools" tools:ignore="MissingTranslation">
+<resources xmlns:tools="http://schemas.android.com/tools">
     <string name="app_version_number">version %1$s</string>
-    <string name="approaching_abbr">1 min</string>
     <string name="arriving_abbr">ARR</string>
     <string name="boarding_abbr">BRD</string>
     <string name="cancelled">Cancelled</string>
     <string name="detour">Detour</string>
     <string name="directionTo">%1$s to</string>
     <string name="feedback_link_form">Send app feedback</string>
     <string name="filterShowAll">All</string>
-    <string name="hello_platform">Hello, %1$s!</string>
-    <string name="icon_description_external_link">External Link</string>
-    <string name="minutes_abbr">%1$s min</string>
+    <string name="icon_description_external_link" tools:ignore="MissingTranslation">External Link</string>
+    <string name="minutes_abbr" tools:ignore="MissingTranslation">%1$s min</string>
```

Comment:
There's no "External Link" on iOS (that icon has no VoiceOver text, and VoiceOver already reads them as "<link text>, button, link"), and the iOS English string is `**%ld** min` but as mentioned in the PR body inline formatting is going to take some work of its own.